roles/ceph-mon/tasks/start_monitor.yml

```diff
@@ -46,6 +46,14 @@
     ansible_distribution != "Ubuntu" and
     not is_ceph_infernalis
 
+- name: enable systemd unit file for mon instance (for or after infernalis)
```

Comment:
could you please use the yaml syntax that we follow?
so this will give you:

````
file:
  src: /usr/lib/systemd/system/ceph-mon@.service
  dest: /etc/systemd/system/multi-user.target.wants/ceph-mon@{{ ansible_hostname }}.service
  state: link
```

Thanks!